### ISSUE TITLE
extensions are hardcoded in constructor

### ISSUE THREAD

**Issue by liitfr** [carrot/roots-browserify#28]
Hello !

In index.coffee, I read :

```
      @b = browserify
        entries: @files
        extensions: ['.js', '.json', '.coffee']
        debug: opts.sourceMap
        cache: @cache
        packageCache: @pkg_cache
```

shouldn't it use opts.opts.extensions ?

See you. Mat

Edit : I will add it in my PR !

**Comment by jescalan** [comment]
Definitely, good catch! Want to send in a PR to fix this, get the first contribution under your belt? 😁

**Comment by liitfr** [comment]
fixed in #29
